# time parameter

mov_time : 3.0		# movement time

# target pose [deg]

tar_pose :
  1 : -32.60		# r_arm_sh_p1
  2 : 32.60		# l_arm_sh_p1
  3 : -59.11		# r_arm_sh_r
  4 : 59.11		# l_arm_sh_r
  5 : 6.61		# r_arm_sh_p2
  6 : -6.61		# l_arm_sh_p2
  7 : -28.33		# r_arm_el_y
  8 : 28.33		# l_arm_el_y
  9 : -59.65		# r_arm_wr_r
  10 : 59.65		# l_arm_wr_r
  11 : 9.02		# r_arm_wr_y
  12 : -9.02		# l_arm_wr_y
  13 : -4.88		# r_arm_wr_p
  14 : 4.88		# l_arm_wr_p
  15 : 0.0		# r_leg_hip_y
  16 : 0.0		# l_leg_hip_y
  17 : 0.0		# r_leg_hip_r
  18 : 0.0		# l_leg_hip_r
  19 : 0.0		# r_leg_hip_p	
  20 : 0.0		# l_leg_hip_p
  21 : 0.0		# r_leg_kn_p
  22 : 0.0		# l_leg_kn_p
  23 : 0.0		# r_leg_an_p
  24 : 0.0		# l_leg_an_p
  25 : 0.0		# r_leg_an_r
  26 : 0.0		# l_leg_an_r
  27 : 0.0		# torso_y
  28 : 0.0		# head_y
  29 : 0.0		# head_p
  30 : 3.0		# l_arm_grip
  31 : 3.0		# r_arm_grip
